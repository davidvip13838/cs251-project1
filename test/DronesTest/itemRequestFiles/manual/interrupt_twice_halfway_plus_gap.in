5 4
5 10 11 11 1000
