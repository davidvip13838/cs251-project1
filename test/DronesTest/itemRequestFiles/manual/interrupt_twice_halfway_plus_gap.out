3 15
2 23
1 31
0 37
4 1008
